.bin/allspark

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Good I think that that is not need to be hidden you can to create a file called allspark on root of project and get a node shebang like that and should works well
```bash
#!/usr/bin/node
```

Reply:
Done